StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -195,7 +196,9 @@ public final class EmbeddedPaymentElement {
     internal private(set) var embeddedPaymentMethodsView: EmbeddedPaymentMethodsView
     internal private(set) var loadResult: PaymentSheetLoader.LoadResult
     internal private(set) var latestUpdateTask: Task<UpdateResult, Never>?
-    private let analyticsHelper: PaymentSheetAnalyticsHelper
+    internal let analyticsHelper: PaymentSheetAnalyticsHelper
+    internal lazy var paymentHandler: STPPaymentHandler = { STPPaymentHandler(apiClient: configuration.apiClient) }()
+    internal var savedPaymentMethods: [STPPaymentMethod]
```

Comment:
I'm wary of having two sources of truth for savedPaymentMethods - this and `loadResult.savedPaymentMethods`.  Maybe we should remove `var loadResult` and replace with the `var intent, var elementSession` etc.

Reply:
Just want to verify -- by removing loadResult and replacing with intent & elementSession, it implies making an initial copy of the saved payment methods from the load result into the proposed `var savedPaymentMethods`, then updating it as we add/remove/update payment methods?

Reply:
Yeah that's the idea. Don't feel strongly, maybe it's worse in some other way?

Reply:
I think your original proposal of storing savedPaymentMethods & elementsSessions separately is better, as I just noticed that the `update()` code overrides the loadResult.  Instead of `update()` setting `loadResult` we can have `update()` directly set `savedPaymentMethods`, `elementsSessions` and any other items that come from loadResult.